src/options.js

```diff
@@ -59,7 +59,7 @@ const translateOptions = (options = {}) => {
   };
   return options;
```

Comment:
@erikjung YESSS, I already had been given pause by mutating the passed options. Let me futz with this idea...